python/amici/setup.template.py

```diff
@@ -32,7 +33,17 @@ def getAmiciLibs():
             #'klu', 'colamd', 'btf', 'amd', 'suitesparseconfig'
             ]
 
-cxx_flags = ['-std=c++11']
+compiler_name = distutils.ccompiler.get_default_compiler()
+if compiler_name == 'mingw32':
```

Comment:
Is "mingw32" a possible return value for you? My distutils knows only about "unix" and "msvc".
Therefore, I think that this might break things for mingw users. Could you please check that with selecting mingw via `CC` environment variable as described in the Windows installation guide?